### DISCUSSION TITLE
Release 1.2

### DISCUSSION THREAD

**Discussion by warrensbox** [warrensbox/terraform-switcher#442] [Releases]
* #429 
* #435 
* #437

**Comment by warrensbox** [comment]
I'd like to propose that we merge PR #435 (which includes #429), and then merge #437. Please let me know what you think. 
@MatrixCrawler @MatthewJohn 

@yermulnik and I have reviewed both PRs and provided comments. I believe we have a resolution for the TOML file as well. 

I would like to set the **Release 1.2** date to **Friday Jun 7.** We can release an Alpha version if merged everything to master before that date.

**Reply by MatrixCrawler** [reply]
Sounds good to me
